config/config.go

```diff
@@ -154,10 +155,10 @@ type dbtProject struct {
 	Seeds   map[string]map[string]interface{} `yaml:"seeds"`  // "Seeds[project_name][key]value"
 }
 
-func readDBTProject() (dbtProject, error) {
+func readDBTProject(customConfigPath string) (dbtProject, error) {
 	project := dbtProject{}
 
-	bytes, err := ioutil.ReadFile("dbt_project.yml")
+	bytes, err := ioutil.ReadFile(customConfigPath + "dbt_project.yml")
```

Comment:
Optional: something like customConfigPath.RightTrim("/")+"/dbt_project.yml" might be useful ( although you'll have to treat the empty case...

Reply:
Handled here
https://github.com/monzo/ddbt/pull/50/commits/fe62948c2a9e52807027a9c931254a75f0fea833

Reply:
👍